migrations/Version20240729132206.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20240729132206 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Migrate legacy string values to strings that matches enum case defined in IssueStatusEnum.php';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql("
+            UPDATE issue SET status =
+            CASE
+                WHEN status = 'Lukket' THEN 'done'
```

Comment:
what about the statuses from leantime? -1...4

Reply:
Good point. I added them!